app.json

```diff
@@ -6,7 +6,8 @@
 
   ],
   "env": {
-    "NPM_CONFIG_PRODUCTION": false
+    "NPM_CONFIG_PRODUCTION": "false",
+    "DATABASE_URL": "postgres://nauqnlwp:TsIEk6q2puSnXc6dtrZQk6LznrvbrLiy@isilo.db.elephantsql.com:5432/nauqnlwp"
```

Comment:
Let's not set our env variable for database url here, this is because the way it is now, anybody that has access to our repo will be able to gain access to our database and do malicious things with it, you should rather set this variable on  Heroku.

Reply:
okay fixing this now